src/modules/app.js

```diff
@@ -0,0 +1,35 @@
+/* eslint no-undefined: "error" */
+
+import ListTemplate from './templateList.js';
+import localGet from './storage.js';
+
+const todoListContainer = document.querySelector('.toDoListItemContainer');
+
+const addList = (description, completed) => {
+  const listAdded = new ListTemplate(description, completed);
+  localGet.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(localGet));
```

Comment:
Kindly note that you need to save the index of TodoTasks as well. As you can see below they are not being saved in the local storage. Please fix that 🤗

![image](https://user-images.githubusercontent.com/73714615/193602272-cf0a6c44-546a-412f-bcad-b8ecad647824.png)